mkdocs.yml

```diff
@@ -49,6 +49,7 @@ nav:
       - Customizable: "features/customization.md"
       - Interoperable: "features/interoperable.md"
       - Powerful: "features/powerful.md"
+      - Moderation: "features/moderation.md"
```

Comment:
I don't think this is a top level item. There are going to get reorganized anyway probably when @fenekku is back, but I would put it under Features awalk-through for now.